src/DeviceInput/InputDevices/webDeviceInputSystem.ts

```diff
@@ -141,7 +141,7 @@ export class WebDeviceInputSystemImpl implements IDeviceInputSystem {
      * @param deviceType Type of device to check for
      * @returns bool with status of device's existence
      */
-    public isDeviceAvailable(deviceType: DeviceType) {
+    public isDeviceAvailable(deviceType: DeviceType): boolean {

```

Comment:
nit: I would recommend adding return types to all of the functions not just this one.

Reply:
added